src/Uno.UI/UI/Xaml/Controls/TextBox/OverlayTextBoxView/OverlayTextBoxViewExtension.skia.cs

```diff
@@ -165,6 +167,51 @@ public void UpdatePosition()
 		}
 	}
 
+	public void UpdateClip()
+	{
+		var clip = CalculateClipping(_contentElement.FindFirstChild<TextBlock>(), _contentElement.GetParent() as FrameworkElement);
+		_textBoxView?.SetClip((int)clip.clippingLeft, (int)clip.clippingRight, (int)clip.clippingTop, (int)clip.clippingBottom);
+
+		if (_lastSize.Width < clip.clippingLeft + clip.clippingRight || _lastSize.Height < clip.clippingTop + clip.clippingBottom)
+		{
+			_textBoxView?.SetVisibility(false);
+		}
+		else
+		{
+			_textBoxView?.SetVisibility(true);
+			if (_owner.TextBox?.FocusState != FocusState.Unfocused)
+			{
+				_textBoxView?.SetFocus();
+			}
+		}
```

Comment:
I haven't verified on Windows, but I think the TextBox could still have focus even if not currently shown, and typing could still work (and could even bring the TextBox to view)